logos/logos-ui/src/styles/_glass.scss

```
@@ -1,8 +1,7 @@
-// ─── Glass System ─────────────────────────────────────────────────────────────
+// ─── Card/Panel Surface System ────────────────────────────────────────────────
 //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)